rd-kt/rd-framework/src/commonMain/kotlin/com/jetbrains/rd/framework/TaskInterfaces.kt

```diff
@@ -95,11 +92,15 @@ val <T> IRdTask<T>.isSucceeded : Boolean get() = result.valueOrNull is RdTaskRes
 val <T> IRdTask<T>.isCanceled : Boolean get() = result.valueOrNull is RdTaskResult.Cancelled
 val <T> IRdTask<T>.isFaulted : Boolean get() = result.valueOrNull is RdTaskResult.Fault
 
+interface ILifetimedRdCall<in TReq, out TRes> {
```

Comment:
This interface introduce new complexity. Why can't you mix everything into IRdCall ?

Reply:
I agree. But we wanted to prohibit the use of the "Start" method for Bindable Tasks without Lifetime. 
Optionally, we can remove (or at least deprecate) the `start` method without lifetime. Then we won't need separate interfaces. What do you think about that?

Reply:
Maybe we can achieve this goal by throwing exception from Start() method if lifetime isn't passed and return value is bindable.

Reply:
I don't think it's a better solution to throw an exception in runtime if we can disable calling the `start` method without lifetime at compile level.
Do we really need the `start` method without lifetime. I think the best solution is to remove (mark as Deprecated) this method.